Backend/FastAPI/routes/grok.py

```
@@ -0,0 +1,81 @@
+import os
+from dotenv import load_dotenv
+from groq import Groq  # Groq SDK
+from langchain_core.prompts import PromptTemplate
```

Ruff (0.12.2)

`langchain_core.prompts.PromptTemplate` imported but unused

Remove unused import: `langchain_core.prompts.PromptTemplate`

(F401)